src/actions/index.js

```diff
@@ -0,0 +1,86 @@
+export const GET_MOVIE_LIST = 'GET_MOVIE_LIST';
+export const GET_SORT_LIST = 'GET_SORT_LIST';
+export const GET_FILTER_LIST = 'GET_FILTER_LIST';
+
+export const ADD_MOVIE = 'ADD_MOVIE';
+export const EDIT_MOVIE = 'EDIT_MOVIE';
+export const DELETE_MOVIE = 'DELETE_MOVIE';
+
+export const START_ADD_MOVIE = 'START_ADD_MOVIE';
+export const START_EDIT_MOVIE = 'START_EDIT_MOVIE';
+export const CANCEL_EDIT_MOVIE = 'CANCEL_EDIT_MOVIE';
+
+export const SHOW_DETAIL_MOVIE = 'SHOW_DETAIL_MOVIE';
+export const CANCEL_DETAIL_MOVIE = 'CANCEL_DETAIL_MOVIE';
+
+export const SELECT_SORT = 'SELECT_SORT';
+export const SELECT_FILTER = 'SELECT_FILTER';
+
+// -------------------------------------------------------
+
+export const getMovieListAction = () => ({
```

Comment:
this is not an action, it's an action creator - a function that returns an action a plain javascript object

Reply:
createMovieListAction